### PR TITLE
CoverBrowser: fix file popup dialog

### PR DESCRIPTION
**Pull request by hius07**
Fixes https://github.com/koreader/koreader/pull/11437#issuecomment-2008321927.
First discussion: https://github.com/koreader/koreader/pull/11404#discussion_r1465954851

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/11576)

### CODE REVIEW

File: plugins/coverbrowser.koplugin/covermenu.lua [hunk 1]
```diff
@@ -232,7 +232,9 @@ function CoverMenu:updateItems(select_number)
     -- we replace it by ours.
     -- (FileManager may replace file_chooser.showFileDialog after we've been called once, so we need
     -- to replace it again if it is not ours)
-    if self.showFileDialog and self.showFileDialog ~= self.showFileDialog_ours then
+    if self.path -- FileManager only
+        and (not self.showFileDialog_ours -- never replaced
+              or self.showFileDialog ~= self.showFileDialog_ours) then -- it is no more ours
```
Comment (Frenzie) [inline plugins/coverbrowser.koplugin/covermenu.lua:237]: no longer :-p

**Review by Frenzie — APPROVED** [review]

File: plugins/coverbrowser.koplugin/covermenu.lua (hunk 1 above)
Comment (hius07) [reply plugins/coverbrowser.koplugin/covermenu.lua:237]: Restored original wordings from https://github.com/koreader/koreader/pull/2940.

**Review by poire-z — APPROVED** [review]

File: plugins/coverbrowser.koplugin/covermenu.lua (hunk 1 above)
Comment (Frenzie) [reply plugins/coverbrowser.koplugin/covermenu.lua:237]: It is a way @poire-z likes to put it. ;-P Anyway, as stated merging is fine by me, but it should be "no longer" ^_^

Comment (poire-z) [reply plugins/coverbrowser.koplugin/covermenu.lua:237]: (Yes, my English is a bit unstable on "no more" vs. "no longer" :))

**Pull request merged**